### PR TITLE
[FIF-213] Fix to add ETLProject to Kotlin to create subproject

### PR DESCRIPTION
**Pull request by kurtjohnson-HS**
Need to add ETL as subproject.

### CODE REVIEW

**Comment by kurtjohnson-HS** [comment]
@stephenfuqua I fixed a lint issue that arose from the TeamCity runs. I also noticed that the service was failing to properly monitor postgres because the service's XML had an improper Postgres protocol URL for the connection string ENVVAR. Those changes make the ETL build for regular branches and PRs.

File: EdFi.Buzz.Etl/eng/Windows/EdfiBuzzEtl.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <id>EdFi-Buzz-ETL</id>
     <name>Ed-Fi Buzz Project Survey ETL</name>
     <description>Runs Node.js as a service to host the Ed-Fi Buzz Survey Loader</description>
-    <env name="DATABASE_URL" value="postgres://{1}:{2}@{3}:{4}/{5}" />
+    <env name="DATABASE_URL" value="postgresql://{1}:{2}@{3}:{4}/{5}" />
```
Comment (stephenfuqua) [inline EdFi.Buzz.Etl/eng/Windows/EdfiBuzzEtl.xml:5]: I've never seen a database url like that.

Comment (stephenfuqua) [reply EdFi.Buzz.Etl/eng/Windows/EdfiBuzzEtl.xml:5]: ignore. just musing out loud.

**Comment by stephenfuqua** [comment]
Looks good, I think it is ready to go.

**Pull request merged**